src/Catalyst.Protocol/Wire/TransactionBroadcast.cs

```diff
@@ -42,6 +44,59 @@ public partial class TransactionBroadcast
              && ConfidentialEntries.All(e => e.IsValid());
         }
 
+        /// <summary>
+        /// TODO: discuss OnConstruction - why is it designed this way 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4350015)